modules/apps/transfer/ibc_module.go

```diff
@@ -203,26 +240,26 @@ func (im IBCModule) OnRecvPacket(
 		}
 	}
 
-	eventAttributes := []sdk.Attribute{
-		sdk.NewAttribute(sdk.AttributeKeyModule, types.ModuleName),
-		sdk.NewAttribute(sdk.AttributeKeySender, data.Sender),
+	ctx.EventManager().EmitEvent(sdk.NewEvent(
```

Comment:
I can readjust this back to do a single event emission like before, didn't realize that was the original behaviour here